frontend/src/components/common/RadioImageSelectGroup.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  Box,
+  FormControl,
+  FormLabel,
+  HStack,
+  Image,
+  Text,
+  useRadio,
+  useRadioGroup,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+
+import customTheme from "../../theme";
+import { DonationSizeInterface } from "../pages/Scheduling/types";
+
+interface RadioImageSelectGroupProps {
+  value: string;
+  values: DonationSizeInterface[];
+  name: string;
+  isRequired?: boolean;
+  onChange: (arg0: any) => void;
```

Comment:
small suggestion but I think the right typing should be: 

```typescript
import { StringOrNumber } from "@chakra-ui/utils";
```
```suggestion
  onChange: (arg0: arg0: React.ChangeEvent<HTMLInputElement> | StringOrNumber,) => void;
```